phylogenetic/bin/newreference.py

```diff
@@ -6,10 +6,12 @@
 
 def new_reference(referencefile, outgenbank, outfasta, gene):
     ref = SeqIO.read(referencefile, "genbank")
+    startofgene = None
+    endofgene = None
     for feature in ref.features:
         if feature.type == 'source':
             ref_source_feature = feature
-        if feature.type =='gene':
+        if feature.type =='gene' or feature.type == 'CDS':
```

Comment:
It'd be preferable for our workflows to use core Augur functions (e.g. `augur.utils.load_features`) for this kind of parsing. I realize this script is copied from RSV so don't consider this blocking but I think it'd be an improvement.

Reply:
I agree and thanks for flagging, I hope we can use a core Augur function in a future PR :)

Reply:
Just noting that this comment and https://github.com/nextstrain/rsv/pull/55#discussion_r1518248004 spawned a draft PR for augur: 

* https://github.com/nextstrain/augur/pull/1435 

Not urgent since we have a working solution.